src/__test__/index.test.ts

```diff
@@ -2,48 +2,66 @@ const ServerlessOfflineSSM = require('../index')
 
 jest.mock('../util', () => ({
   getVarsFromEnv: jest.fn().mockReturnValue({
-    'test': 'some value',
-  })
+    test: 'some value',
+  }),
 }))
 
 describe('serverless-offline-ssm', () => {
   const serverlessMock = () => ({
     version: '1.52.0',
     processedInput: { commands: ['offline'] },
     service: {
-      custom: {}
+      custom: {},
     },
     variables: {
       ssmRefSyntax: RegExp(/(test)/),
-      variableResolvers: [{
-          serviceName: 'SSM'
-        }
-      ]
-    }
+      variableResolvers: [
+        {
+          serviceName: 'SSM',
+        },
+      ],
+    },
   })
 
   describe('offline', () => {
     test('should initialize when using serverless offline', () => {
       const instance = new ServerlessOfflineSSM(serverlessMock())
       expect(instance.serverless.variables.getValueFromSsmOffline).toBeDefined()
-      expect(instance.serverless.variables.variableResolvers[0].resolver).toBeDefined()
+      expect(
+        instance.serverless.variables.variableResolvers[0].resolver,
+      ).toBeDefined()
+    })
+
+    test('should initialize when using invoke local', () => {
```

Comment:
I think vscode prettier was picked up and update the whole file. I've just added this test ^^